DolbyIO/Source/Private/DolbyIOVideoSink.cpp

```diff
@@ -73,6 +73,16 @@ namespace DolbyIO
 			          });
 			return Ret;
 		}
+
+		void UnbindMaterialImpl(UMaterialInstanceDynamic& Material)
+		{
+			static UTexture2D* EmptyTexture = nullptr;
```

Comment:
simpler and the same semantic will have: `static UTexture2D* EmptyTexture = CreateEmptyTexture();`

Reply:
Fair enough. I don't know why I kept it this way :D